src/goto-instrument/contracts/contracts.cpp

```diff
@@ -473,18 +473,19 @@ code_contractst::create_ensures_instruction(
 }
 
 bool code_contractst::apply_function_contract(
-  goto_programt &goto_program,
-  goto_programt::targett target)
+  const irep_idt &function,
+  goto_programt &function_body,
+  goto_programt::targett &target)
```

Comment:
Could this be made `const`:

```suggestion
  const goto_programt::targett &target)
```

I can see that we might be modifying the `function_body`, but is the `target` modified too?

Reply:
Target is used as an iterator by sub-calls and might be modified using operators `++` or `--`.

Reply:
Thanks, resolving this then.